pkg/integrations/blackbox_exporter/blackbox.go

```diff
@@ -0,0 +1,23 @@
+package blackbox_exporter
+
+import (
+	"net/http"
+
+	"github.com/go-kit/log"
+	blackbox_config "github.com/prometheus/blackbox_exporter/config"
+	"github.com/prometheus/blackbox_exporter/prober"
+)
+
+type blackboxHandler struct {
+	cfg     *Config
+	modules *blackbox_config.Config
+	log     log.Logger
+}
+
+var (
+	probeTimeoutOffset = 0.5
+)
+
+func (bbh *blackboxHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	prober.Handler(w, r, bbh.modules, bbh.log, &prober.ResultHistory{}, probeTimeoutOffset, nil)
+}
```

Comment:
It seems like you could avoid having to define blackboxHandler if you use http.HandlerFunc instead in `(*Integration).MetricsHandler`